cmd/kops/create_cluster.go

```diff
@@ -70,6 +70,9 @@ type CreateClusterOptions struct {
 
 	// Enable/Disable Bastion Host complete setup
 	Bastion bool
+
+	NgwIds  string
```

Comment:
Just curious, why are we adding to members here? It doesn't look like we are adding any more command line flags.. (which is good).. but wondering the thought behind this?

Also I know we have `VPCID` already, but do we really want to be adding AWS specific members here? Maybe a discussion for GCE porting later.. but figured I would ask